source/Ship.h

```diff
@@ -473,6 +486,8 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	// int swizzle;
 	// const Government *government;
 
+	std::string activity;
+
```

Comment:
this one is a bit out of place, we maybe want to move it somewhere to the shield/crew stats

Reply:
OK, How about down at Line 553, where it's right after the lines dealing with commands, personalities, and right before the sectio non outfits and attributes and the like?

Reply:
moved.